packages/core/src/browser/shell/tab-bar-decorator.ts

```diff
@@ -0,0 +1,88 @@
+/********************************************************************************
+ * Copyright (C) 2019 Ericsson and others.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License v. 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ *
+ * This Source Code may also be made available under the following Secondary
+ * Licenses when the conditions for such availability set forth in the Eclipse
+ * Public License v. 2.0 are satisfied: GNU General Public License, version 2
+ * with the GNU Classpath Exception which is available at
+ * https://www.gnu.org/software/classpath/license.html.
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
+ ********************************************************************************/
+
+import { inject, injectable, named, postConstruct } from 'inversify';
+import { Event, Emitter, Disposable, DisposableCollection, ContributionProvider } from '../../common';
+import { Title, Widget } from '@phosphor/widgets';
+import { WidgetDecoration } from '../widget-decoration';
+
+export const TabBarDecorator = Symbol('TabBarDecorator');
+
+export interface TabBarDecorator {
+
+    /**
+     * The unique identifier of the tab bar decorator.
```

Comment:
There is inconsistencies between the naming.
Here we refer to `tab bar` while on line 32 we use `tabbar`.